packages/clerk-js/src/ui/components/OrganizationList/OrganizationListPage.tsx

```diff
@@ -0,0 +1,214 @@
+import { useState } from 'react';
+
+import { useCoreOrganizationList, useEnvironment, useOrganizationListContext } from '../../contexts';
+import { Box, Button, Col, descriptors, Flex, localizationKeys, Spinner } from '../../customizables';
+import { Card, CardAlert, Divider, Header, useCardState, withCardStateProvider } from '../../elements';
+import { useInView } from '../../hooks';
+import { CreateOrganizationForm } from '../CreateOrganization/CreateOrganizationForm';
+
+import { PreviewListItems, PreviewListSpinner } from './shared';
+import { InvitationPreview } from './UserInvitationList';
+import { MembershipPreview, PersonalAccountPreview } from './UserMembershipList';
+import { SuggestionPreview } from './UserSuggestionList';
+import { organizationListParams } from './utils';
+
+const useFetchList = () => {
```

Comment:
🙃  Nitpicking comment: 

Can we rename this one to something more relevant? Like, `useCoreOrganizationListWithRef` or `useCoreOrganizationListInView` or sth else. `useFetchList` sounds a little generic.

Reply:
Makes sense